CorsixTH/Lua/app.lua

```diff
@@ -313,6 +313,39 @@ function App:init()
   return true
 end
 
+---
+-- Initialises a Lua DBGp client and
+-- connects it to a DBGp server so that
+-- it can debug lua scripts at runtime.
+-- @return error_message
```

Comment:
Is this type of function comments recognized by Eclipse? We do have commenting standards tailored towards LDocGen (not entirely consistently used throughout the code unfortunately), but if another standard would be better for us we might consider changing?